packages/devtools_app/lib/src/inspector/flutter/inspector_data_models.dart

```diff
@@ -74,15 +60,50 @@ class LayoutProperties {
 
   LayoutProperties get largestHeightChild => _largestHeightChild;
 
-  double get smallestWidthChildPercentage => _smallestWidthChildPercentage;
+  double get smallestWidthChildPercentage => _smallestWidthChild.width / width;
 
-  double get smallestHeightChildPercentage => _smallestHeightChildPercentage;
+  double get smallestHeightChildPercentage => _smallestHeightChild.height / height;
 
-  double get largestWidthChildPercentage => _largestWidthChildPercentage;
+  double get largestWidthChildPercentage => _largestWidthChild.width / width;
 
-  double get largestHeightChildPercentage => _largestHeightChildPercentage;
+  double get largestHeightChildPercentage => _largestHeightChild.height / height;
 
-  static Constraints deserializeConstraints(Map<String, Object> json) {
+  double get width => size?.width;
+
+  double get height => size?.height;
+
+  Iterable<double> get childrenWidth => children?.map((child) => child.width);
+
+  Iterable<double> get childrenHeight => children?.map((child) => child.height);
+
+  String describeWidthConstraints() {
+    if (constraints.hasBoundedWidth)
+      return describeAxis(constraints.minWidth, constraints.maxWidth, 'w');
+    return 'w=unconstrained';
+  }
+
+  String describeHeightConstraints() {
+    if (constraints.hasBoundedHeight)
+      return describeAxis(constraints.minHeight, constraints.maxHeight, 'h');
+    return 'h=unconstrained';
+  }
+
+  String describeWidth() {
+    return 'w=' + size.width.toStringAsFixed(fractionDigits);
+  }
+
+  String describeHeight() {
+    return 'h=' + size.height.toStringAsFixed(fractionDigits);
+  }
+
+  static const int fractionDigits = 1;
```

Comment:
I think @kenzieschmoll may have already created a helper handling numeric precision. Search for toStringAsFixed in the code base to find it.

Reply:
I couldn't find it, might need more pointer

Reply:
nevermind. Go ahead an add and extra helper to
lib/src/utils.dart 
which has some helpers for string formatting.